docs/rules/idiomatic/no-defined-entrypoint.md

```diff
@@ -0,0 +1,83 @@
+# no-defined-entrypoint
+
+**Summary**: At least one entrypoint should be defined
+
+**Category**: Idiomatic
+
+**Type**: Aggregate - only runs when more than one file is provided for linting
+
+**Avoid**
+```rego
+package policy
+
+import future.keywords.if
+import future.keywords.in
+
+default allow := false
+
+# Nothing wrong with this rule, but an
+# entrypoint should be documented as such
+allow if {
+    some role in input.user.roles
+    role in data.permissions.admin_roles
+}
```

Comment:
Would these examples be clearer if the unnamed entrypoint `allow` showed how entrypoints often gather the results from various other rules?

Reply:
Yeah.. I agree that'd be good! I've wanted to make all examples self-contained, so that they'd be copy-pasteable... but might be worth expanding a bit on this example and have a helper rule or two. I'll take a look!

Reply:
Updated to include two helper rules :)